webdriver/tests/bidi/script/add_preload_script/invalid.py

```diff
@@ -192,3 +193,39 @@ async def test_params_sandbox_invalid_type(bidi_session, sandbox):
         await bidi_session.script.add_preload_script(
             function_declaration="() => {}", sandbox=sandbox
         ),
+
+
+@pytest.mark.parametrize("contexts", [False, 42, '_UNKNOWN_', {}])
+async def test_params_contexts_invalid_type(bidi_session, contexts):
+    with pytest.raises(error.InvalidArgumentException):
+        await bidi_session.script.add_preload_script(
+            function_declaration="() => {}",
+            contexts=contexts
+        ),
+
+async def test_params_contexts_invalid_context(bidi_session):
```

Comment:
Please add tests for a context having an invalid type as well before this test.

This test should be named to follow other test namings:

```suggestion
async def test_params_contexts_context_invalid_value(bidi_session):
```